gtest-parallel

```diff
@@ -346,6 +358,7 @@ if options.format == 'raw':
   pass
 elif options.format == 'filter':
   logger = FilterFormat()
+  logger.zip_file = os.path.join(options.output_dir, 'passing_test_logs.zip')
```

Comment:
How about changing this to create one single log file instead of the zipping, from all job IDs (passing and failing ones). Then maybe also emit separate log files for the interrupted/failed tests.
That would fit the interactive use case as well (assuming people prefer one log file rather than thousands of files). It will also solve the problem on the bot (that we archive thousands of files per test run).